frameworks/g-cloud-9/manifests/download_results.yml

```diff
@@ -0,0 +1,5 @@
+- name: Body
```

Comment:
Why does G9 get a capitalised `Body` but previous frameworks only get a small `body`?  Also, the name could maybe be more descriptive - "Body" doesn't really tell me what this is.

Reply:
I could give it a more descriptive name, I suppose... but the name doesn't ever go anywhere.

I'll give it one anyway since you mentioned it!